examples/with-react-esi/README.md

```diff
@@ -0,0 +1,26 @@
+# React ESI example
+
+## How to use
+
+### Download manually
+
+Download the example:
```

Comment:
This section seems to be missing the `create-next-app` instructions 🤔

Reply:
For the app to be fully functional, a cache server with ESI support is mandatory. It's why I not added the instructions using `create-next-app`. But I can add them back with a not explaining that you also need to setup properly Varnish or whatever having ESI support.